src/vs/base/browser/markdownRenderer.ts

```diff
@@ -190,12 +190,19 @@ export function renderMarkdown(markdown: IMarkdownString, options: MarkdownRende
 
 	element.innerHTML = insane(renderedMarkdown, {
 		allowedSchemes,
+		allowedTags: [
+			'a', 'abbr', 'article', 'b', 'blockquote', 'br', 'caption', 'code', 'del', 'details', 'div', 'em',
+			'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'hr', 'i', 'img', 'ins', 'kbd', 'li', 'main', 'mark',
+			'ol', 'p', 'pre', 'section', 'span', 'strike', 'strong', 'sub', 'summary', 'sup', 'table',
+			'tbody', 'td', 'th', 'thead', 'tr', 'u', 'ul', 'input'
+		],
```

Comment:
This is risky, right? Don't we use this function to also render extension contributed markdown and thereby also allowing them to render html?

Reply:
Thanks for the remind. This one is about allowing the core markdown renderer to display `checkbox` which is sanitized by `insane`. I'll revert the change for now and see ppl's feedback.